### PR TITLE
chore: set node 16 as default

### PR DESCRIPTION
**Pull request by pedrolamas**
Signed-off-by: Pedro Lamas <pedrolamas@gmail.com>

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -73,24 +73,6 @@ module.exports = {
       .use('file-loader')
       .loader('file-loader')
       .end()
-    config.module
-      .rule('monaco-safari-fix')
```
Comment (pedrolamas) [inline vue.config.js:77]: This was a Safari fix introduced for monaco editor, which I believe is not required anymore (was fixed upstream in the monaco components).

**Comment by Biorn1950** [comment]
Excellent idea, there are a lot of security fix since the v14.

**Comment by pedrolamas** [comment]
@Biorn1950 if you have a mac with Safari, please feel free to checkout the branch and test (specifically, the inline file editor), as I removed some code that was fixing issues with Safari as I believe those are no longer required.

**Comment by Biorn1950** [comment]
> @Biorn1950 if you have a mac with Safari, please feel free to checkout the branch and test (specifically, the inline file editor), as I removed some code that was fixing issues with Safari as I believe those are no longer required.

Looks ok.

**Review by Biorn1950 — APPROVED** [review]
LGTM

File: package.json [hunk 2]
```diff
@@ -92,7 +92,6 @@
     "@vue/eslint-config-standard": "^5.1.2",
     "@vue/eslint-config-typescript": "^5.0.2",
     "@vue/test-utils": "^1.2.1",
-    "babel-core": "^7.0.0-bridge.0",
```
Comment (eduncan911) [inline package.json:95]: yeah, it's this "breakage" stuff nodejs packages do that is a pita for these upgrades.  good job!

File: package.json [hunk 3]
```diff
@@ -104,16 +103,15 @@
...
+    "monaco-vscode-textmate-theme-converter": "^0.1.7",
     "sass": "~1.32.8",
     "sass-loader": "^10.2.0",
     "serve": "^11.3.2",
     "shx": "^0.3.3",
     "standard-version": "^9.3.0",
-    "string-replace-loader": "^3.0.2",
     "threads-plugin": "^1.4.0",
-    "ts-jest": "^27.0.3",
+    "ts-jest": "^26.5.6",
```
Comment (eduncan911) [inline package.json:114]: regression?

File: src/components/ui/AppChart.vue [hunk 4]
```diff
@@ -10,7 +10,7 @@
         ref="chart"
         :option="opts"
         :setOptionOps="{ notMerge: true }"
-        :initOpts="{ renderer: 'svg' }"
+        :init-options="{ renderer: 'svg' }"
```
Comment (eduncan911) [inline src/components/ui/AppChart.vue:13]: so they remove camel case, but they leave `setOptionsOps`?  lol

**Review by eduncan911 — APPROVED** [review]
LGTM

File: package.json (hunk 3 above)
Comment (pedrolamas) [reply package.json:114]: Had to go back as npm couldn't correctly resolve the dependencies without using "legacy" mode... which is BAD and should be avoided at all costs!!

File: src/components/ui/AppChart.vue (hunk 4 above)
Comment (pedrolamas) [reply src/components/ui/AppChart.vue:13]: Odd, I know... but without this change, the build was breaking! After that it worked fine!

**Pull request merged**